ntoskrnl/se/tokencls.c

```diff
@@ -16,6 +16,13 @@
 
 /* INFORMATION CLASSES ********************************************************/
 
+/* FIXME: Delete this */
```

Comment:
Is it possible to give a more specific hint **when** this can be deleted? Any specific precondition to be met? Anything must have been implemented anywhere? Where?

Reply:
`<winnt.h>` should provide this definition.